### PR TITLE
feat : doc-site

### PR DESCRIPTION
**Pull request by mikesch803**
1. P1 components
2.  highlight for code snippets

### CODE REVIEW

**Comment by mikesch803** [comment]
> 1. P1 components
> 2. highlight for code snippets

This is the documentation site,  the separate repo. The component repo link is https://github.com/mikesch803/component-library/tree/dev

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "liveServer.settings.port": 5501
+    "liveServer.settings.port": 5502
```
Comment (ApurvaSawant11) [inline .vscode/settings.json]: You can delete this folder before creating a pull request.

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me! UI is also nice. Good work:)

**Review by anik31 — APPROVED** [review]
Good work

**Pull request merged**